tox.ini

```diff
@@ -1,14 +1,16 @@
 [tox]
-envlist = py35-django{22}
+envlist = py{35,38}-django{22,30}
 skipsdist = true
 
 [pytest]
-DJANGO_SETTINGS_MODULE = designer.settings.test
+django_settings_module = designer.settings.test
 testpaths = designer/apps
 
 [testenv]
-deps =
-    django22:  -r requirements/django.txt
-    -r {toxinidir}/requirements/test.txt
-commands =
-    {posargs:python -Wd -m pytest}
+deps = 
+	django22: Django>=2.2,<2.3
```

Comment:
We still want `requirements/django.txt` here, we should fix the script if it changed that.  Looks like the indentation is wrong, too.

Reply:
changed to `requirements/django.txt`.